index.js

```diff
@@ -136,34 +145,84 @@ app.post("/", async (req, res) => {
         );
       }
     } else {
-      // insert the data id is as new entry:
+      // Insert the data id is as new entry:
       databaseConnection.insertOne(formData, () => {
         db.close;
       });
     }
-
+    //Sending the name,punchDetails,IP_address,location,userAgent as massage to slack :
     // An access token (from your Slack app or custom integration - xoxp, xoxb)
-    const token = process.env.SLACK_API_KEY;
+    // const token = process.env.SLACK_API_KEY;
 
-    const web = new WebClient(token);
+    // const web = new WebClient(token);
 
-    // This argument can be a channel ID, a DM ID, a MPDM ID, or a group ID
-    const conversationId = process.env.CONVERSATION_ID;
+    // // This argument can be a channel ID, a DM ID, a MPDM ID, or a group ID
+    // const conversationId = process.env.CONVERSATION_ID;
 
-    await web.chat.postMessage({
-      channel: conversationId,
-      text: `User : ${employees[req.body.id] || "Unknown"} \n ${
-        latestEntry === "Punch In" ? "Punch Out" : "Punch In"
-      } : ${current_time_obj} \n Location: ${results},${req.body.lat},${
-        req.body.long
-      }\n IP Address: ${clientIp} \n User Agent: ${userAgentDetails}`,
-    });
+    // await web.chat.postMessage({
```

Comment:
Why this is commented ?

Reply:
For testing purpose i have commented it ,i forgot to  remove this , i will remove this now.